src/redux/reducers/auth.js

```diff
@@ -0,0 +1,140 @@
+import {
+  LOGIN_SUCCESS,
+  LOGOUT,
+  PHONE_SUCCESS,
+  DATA_FAILED,
+  DATA_LOADING,
+  PROGRESS_SUCCESS,
+  PROGRESS_PREVIOUS,
+  EMAIL_SUCCESS,
+  F_NAME_SUCCESS,
+  L_NAME_SUCCESS,
+  REGISTER_FAILED,
+} from "../actions/types";
+const initialState = {
+  isLoggedIn: false,
+  token: "",
+  userId: "",
+  user: null,
+  isLoading: false,
+  isError: false,
+  isSuccess: false,
+  countProgress: 0,
+  errMsg: null,
+  phone: "",
+  email: "",
+  fname: "",
+  lname: "",
+  register: "",
+};
+
+export const authReducer = (state = initialState, action) => {
+  // alert(JSON.stringify(action));
```

Comment:
also in reducers file only the reducers of the used function use here remove all others